Patches.py

```diff
@@ -2913,3 +2944,5 @@ def patch_ganons_tower_bk_door(rom: Rom, flag):
     var = (0x05 << 6) + (flag & 0x3F)
     bytes = [(var & 0xFF00) >> 8, var & 0xFF]
     rom.write_bytes(0x2EE30FE, bytes)
+
```

Comment:
More arbitrary whitespace changes.

Reply:
There were actually 3 lines of white space here.   Removed them all.